doc/Dependencies.md

```diff
@@ -0,0 +1,27 @@
+Third Party Dependencies
+========================
+
+This document is about dependencies that are required to build applications
+using `libtock-rs`. Dependencies required to run `libtock-rs`' tests (such as
+`make`) are outside the scope of this document.
+
+## Unaudited Required Dependencies
+
+`libtock-rs` has the following required build dependencies, none of which are
+currently audited:
+
+* The Rust toolchain, including
+  [`cargo`](https://github.com/rust-lang/cargo),
+  [`rustc`](https://github.com/rust-lang/rust/tree/master/src/rustc), and
+  [`libcore`](https://github.com/rust-lang/rust/tree/master/src/libcore). The
+  specific toolchain version used is specified by the `rust-toolchain` file at
+  the root of the repository.
+* [`syn`](https://crates.io/crates/syn), pulled in by `libtock_codegen`.
+* [`quote`](https://crates.io/crates/quote), pulled in by `libtock_codegen`.
+* [`proc-macro2`](https://crates.io/crates/proc-macro2), pulled in by
+  `libtock_codegen`.
+
+## Avoiding Optional Dependencies
+
+To avoid pulling in optional dependencies, users should use `libtock-core`
```

Comment:
While this can be true in the future this probably doesn't hold at the moment (the dependencies according to the 3rd party policy are the same).

Reply:
I'm not sure I understand you correctly.

`libtock-rs` has a dependency on `futures` that `libtock-core` does not have: https://github.com/tock/libtock-rs/blob/e40b57fc9566dd04e7863a3ed1e347f4561bb2ce/Cargo.toml#L17

Reply:
`core` ships futures as a transitive dependency so maybe we need to distinguish between direct and transitive dependencies?

Reply:
> `core` ships futures as a transitive dependency

I don't see that at all, it appears that `core` has no dependencies (as I expected): https://github.com/rust-lang/rust/blob/master/src/libcore/Cargo.toml

Reply:
You are correct. Now I learned how `core` can provide `Futures` and the futures crate can depend on core.